app/assets/javascripts/short-form/components/preferenceComponent.js.coffee

```diff
@@ -49,6 +50,20 @@ angular.module('dahlia.components')
       @onChange = =>
         @resetPreference()
 
+      # For 588 Mission preference, we need to override some defaults.
+      @is588 = =>
```

Comment:
I might change this to be `isEmploymentDisability` so that it's more tied to the preference than the listing